### ISSUE TITLE
Unable to resolve var: cider.nrepl/wrap-undef

### ISSUE THREAD

**Issue by matthias-margush** [liquidz/vim-iced#94]
Hi. I'm currently getting this error running `iced repl` in a fresh `lein new hello-world`. I don't have any `~/.lein/profiles.clj`.

```
Exception in thread "main" Syntax error compiling var at (/private/var/folders/pt/0w3fhml53wd24yhxn4rxpj7r0000gn/T/form-init4596740160835148845.clj:1:6545).
        at clojure.lang.Compiler.analyzeSeq(Compiler.java:7114)
        at clojure.lang.Compiler.analyze(Compiler.java:6789)
        at clojure.lang.Compiler.analyze(Compiler.java:6745)
        at clojure.lang.Compiler$InvokeExpr.parse(Compiler.java:3888)
        at clojure.lang.Compiler.analyzeSeq(Compiler.java:7108)
        at clojure.lang.Compiler.analyze(Compiler.java:6789)
        at clojure.lang.Compiler.analyze(Compiler.java:6745)
        at clojure.lang.Compiler$InvokeExpr.parse(Compiler.java:3888)
        at clojure.lang.Compiler.analyzeSeq(Compiler.java:7108)
        at clojure.lang.Compiler.analyze(Compiler.java:6789)
        at clojure.lang.Compiler.access$300(Compiler.java:38)
        at clojure.lang.Compiler$LetExpr$Parser.parse(Compiler.java:6384)
        at clojure.lang.Compiler.analyzeSeq(Compiler.java:7106)
        at clojure.lang.Compiler.analyze(Compiler.java:6789)
        at clojure.lang.Compiler.analyze(Compiler.java:6745)
        at clojure.lang.Compiler$BodyExpr$Parser.parse(Compiler.java:6120)
        at clojure.lang.Compiler$FnMethod.parse(Compiler.java:5467)
        at clojure.lang.Compiler$FnExpr.parse(Compiler.java:4029)
        at clojure.lang.Compiler.analyzeSeq(Compiler.java:7104)
        at clojure.lang.Compiler.analyze(Compiler.java:6789)
        at clojure.lang.Compiler.eval(Compiler.java:7173)
        at clojure.lang.Compiler.eval(Compiler.java:7166)
        at clojure.lang.Compiler.eval(Compiler.java:7166)
        at clojure.lang.Compiler.eval(Compiler.java:7166)
        at clojure.lang.Compiler.load(Compiler.java:7635)
        at clojure.lang.Compiler.loadFile(Compiler.java:7573)
        at clojure.main$load_script.invokeStatic(main.clj:452)
        at clojure.main$init_opt.invokeStatic(main.clj:454)
        at clojure.main$init_opt.invoke(main.clj:454)
        at clojure.main$initialize.invokeStatic(main.clj:485)
        at clojure.main$null_opt.invokeStatic(main.clj:519)
        at clojure.main$null_opt.invoke(main.clj:516)
        at clojure.main$main.invokeStatic(main.clj:598)
        at clojure.main$main.doInvoke(main.clj:561)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.lang.Var.applyTo(Var.java:705)
        at clojure.main.main(main.java:37)
Caused by: java.lang.RuntimeException: Unable to resolve var: cider.nrepl/wrap-undef in this context
        at clojure.lang.Util.runtimeException(Util.java:221)
        at clojure.lang.Compiler$TheVarExpr$Parser.parse(Compiler.java:720)
        at clojure.lang.Compiler.analyzeSeq(Compiler.java:7106)
        ... 36 more
```

**Comment by liquidz** [comment]
@matthias-margush Are you sure that  you are using Leiningen 2.9.0?
Leiningen 2.9.0 is using nrepl 0.6.0, and cider-nrepl 0.20.0 is not supporting nrepl 0.6.0.

I'm working to support cider-nrepl 0.21.0 on #91, so could you use Leiningen 2.8.3 until fix it?

**Comment by matthias-margush** [comment]
Ah okay: yes my lein updated itself. Thanks!

**Comment by liquidz** [comment]
@matthias-margush I released vim-iced 0.10.0 (#93)

**Comment by matthias-margush** [comment]
Works great!
